lib/PuppeteerSharp.Tests/Page/AddStyleTagsTests.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class AddStyleTagsTests : PuppeteerPageBaseTest
+    {
+        private const string BackgroundColorScript =
+            "window.getComputedStyle(document.querySelector('body')).getPropertyValue('background-color')";
+
+        [Fact]
+        public async Task ShouldWorkWithAUrl()
+        {
+            await Page.GoToAsync(TestConstants.EmptyPage);
+
+            var styleHandle = await Page.AddStyleTagAsync(new AddTagOptions
+            {
+                Url = "/injectedstyle.css"
+            });
+            
+            Assert.NotNull(styleHandle.AsElement());
+
+            var bgColor = await Page.EvaluateExpressionAsync(BackgroundColorScript);
```

Comment:
use `EvaluateExpressionAsync<string>` and remove the `ToString` on the next line